### PR TITLE
adding `%jupysql`/`%%jupysql` as alias

### LINKED ISSUE #102: Using jupysql magic %sql in databricks
I have been using the sql magic `%sql` using jupysql module a lot in my local computer. 
Nowadays, I am starting to use the databricks and porting all my jupyter notebooks to databricks.

The databricks already have their implementation of `%sql` and thus, this magic from jupySQL may not work.

Is there any way if can use both `%sql` from databricks and from jupysql?

Such as can use some alias of `%sql` such as `%sqljupy`?

### PR DESCRIPTION
**Pull request by edublancas**
closes #102

----
:books: Documentation preview :books:: https://jupysql--112.org.readthedocs.build/en/112/

### CODE REVIEW

File: doc/howto.md [hunk 1]
```diff
@@ -11,6 +11,19 @@ kernelspec:
   name: python3
 ---
 
+```{code-cell} ipython3
+:tags: [hide-cell]
+
+# clean up all .db files (this cell will not be displayed in the docs)
```
Comment (yafimvo) [inline doc/howto.md:17]: According to the comment this cell shouldn't be displayed in the docs but [hide-cell] hides the cell and the user can reveal it.

We can remove this cell with [remove-cell]. [remove an entire code cell](https://jupyterbook.org/en/stable/interactive/hiding.html#remove-an-entire-code-cell)

File: doc/howto.md [hunk 2]
```diff
@@ -230,3 +243,36 @@ Close by passing the alias:
...
+some_engine = create_engine("sqlite:///some.db")
+```
+
+```{code-cell} ipython3
+%sql some_engine
+```
+
++++ {"tags": []}
+
+## Use `%sql`/`%%sql` in databricks
```
Comment (yafimvo) [inline doc/howto.md:261]: Databricks

**Review by yafimvo — CHANGES_REQUESTED** [review]
added minor notes

**Review by tonykploomber — DISMISSED** [review]
LG

**Pull request merged**